validate/validate.go

```diff
@@ -337,11 +337,11 @@ func (v *Validator) CheckRlimits() (msgs []string) {
 func supportedMountTypes(OS string, hostSpecific bool) (map[string]bool, error) {
 	supportedTypes := make(map[string]bool)
 
-	if OS != "linux" && OS != "windows" {
+	if OS != "linux" && OS != "solaris" {
 		logrus.Warnf("%v is not supported to check mount type", OS)
 		return nil, nil
-	} else if OS == "windows" {
-		supportedTypes["ntfs"] = true
+	} else if OS == "solaris" {
+		supportedTypes["lofs"] = true
```

Comment:
On the `type` of `solaris` support, I found the following sentence in the relevant documents:
> Used to specify the FSType on which to operate. The FSType must be specified or must be determinable from /etc/vfstab, or by consulting /etc/default/fs or /etc/dfs/fstypes. 

 But I did not find these documents, only given examples given a support `type`.